src/current/v24.1/configure-logs.md

```diff
@@ -217,7 +217,7 @@ Along with the [common sink parameters](#common-sink-parameters), each Fluentd s
 | `address` | Network address and port of the log collector.                                                                     |
 | `net`     | Network protocol to use. Can be `tcp`, `tcp4`, `tcp6`, `udp`, `udp4`, `udp6`, or `unix`.<br><br>**Default:** `tcp` |
-| `buffering`     | Configures buffering of log messages for the sink, with the following sub-parameters:<br><br><ul><li>`max-staleness`: The maximum time a log message will wait in the buffer before a flush is triggered. Set to `0` to disable flushing based on elapsed time. Default: `5s`</li><li>`flush-trigger-size`: The number of bytes that will trigger the buffer to flush. Set to `0` to disable flushing based on accumulated size. Default: `1MiB`</li><li>`max-buffer-size`: The maximum size of the buffer: new log messages received when the buffer is full cause older messages to be dropped. Default: `50MiB`</li></ul>When `max-staleness` and `flush-trigger-size` are used together, whichever is reached first will trigger the flush. `buffering` is enabled by default for [Fluentd-compatible](#output-to-fluentd-compatible-network-collectors) log sinks. To explicitly disable log buffering, specify `buffering: NONE` instead. This setting is typically disabled for [security-related logs]({% link {{ page.version.version }}/logging-use-cases.md %}#security-and-audit-monitoring). See [Log buffering](#log-buffering-for-network-sinks) for more details and usage.|
+| `buffering` <a name="buffering"></a>     | Configures buffering of log messages for the sink, with the following sub-parameters:<br><br><ul><li>`max-staleness`: The maximum time a log message will wait in the buffer before a flush is triggered. Set to `0` to disable flushing based on elapsed time. Default: `5s`</li><li>`flush-trigger-size`: The number of bytes that will trigger the buffer to flush. Set to `0` to disable flushing based on accumulated size. Default: `1MiB`</li><li>`max-buffer-size`: The maximum size of the buffer: new log messages received when the buffer is full cause older messages to be dropped. Default: `50MiB`</li></ul>When `max-staleness` and `flush-trigger-size` are used together, whichever is reached first will trigger the flush. `buffering` is enabled by default for [Fluentd-compatible](#output-to-fluentd-compatible-network-collectors) log sinks. To explicitly disable log buffering, specify `buffering: NONE` instead. This setting is typically disabled for [security-related logs]({% link {{ page.version.version }}/logging-use-cases.md %}#security-and-audit-monitoring). See [Log buffering](#log-buffering-for-network-sinks) for more details and usage.|
```

Comment:
IIRC, I don't believe we want to publicly advertise this config option until the WAL failover feature has been put through its paces more.

@jbowens thoughts on whether we should advertise this new config option?

Reply:
That makes sense.  @mwang1026 also told me (approximately) that we only want to document the logging stuff in the WAL failover section of the `cockroach start` docs, _not_ in the logging docs.  Maybe that message from him was after I posted this PR and I forgot to update the PR

Therefore I'm super ok with removing the link to this option from the `cockroach start` docs and removing this anchor as well since those changes were the only thing pointing to this anchor.

does that sound good?

Reply:
I'm good with anything for 24.1 for the public preview!

Reply:
I've gone ahead and removed all the links from the WAL failover stuff to the logging flags for now since this is preview.  We can always add more stuff later